src/location/src/location-service/app.py

```diff
@@ -52,6 +84,17 @@ def index():
 def get_locations():
     return jsonify(locations)
 
+@app.route('/locations')
+def get_locations_by_beer():
+    beer_name = request.args.get('beerName')
```

Comment:
For the URL I suggested, you can change this 3 lines to:

```python
@app.route('/locations/beer/<beer_name>')
def get_locations_by_beer(beer_name):
```

https://flask.palletsprojects.com/en/2.0.x/quickstart/#variable-rules